src/plugins/elasticsearch/lib/kibana_index_mappings.js

```diff
@@ -6,5 +6,13 @@ export const mappings = {
         index: 'not_analyzed'
       }
     }
+  },
+  server: {
+    properties: {
+      uuid: {
+        type: 'string',
```

Comment:
In other words, I think this should be `type: 'keyword'` and you shouldn't pass `index` at all.